src/main/python/rlbot/version.py

```diff
@@ -15,6 +15,7 @@
     - Making the details section of bot config files more visible to python GUIs. - tarehart
     - Improved stability of the SetupManager. - DomNomNom
     - Added rendering capability to training exercises. - DomNomNom
+    - Clear the screen when bots retire. - DomNomNom
```

Comment:
The version number also needs to be bumped, otherwise pypi will reject us. I'll go ahead and do that.